sdks/csharp/graph_operations_other.cs

```diff
@@ -1,20 +1,22 @@
 // ------------------ CREATE RELATIONSHIP (Short) ---------------------
 // <CreateRelationship_short>
-BasicRelationship rel = new BasicRelationship();
-rel.TargetId = "myTargetTwin";
-rel.Name = "contains"; // a relationship with this name must be defined in the model
+var rel = new BasicRelationship
+{
+    TargetId = "myTargetTwin",
+    Name = "contains", // a relationship with this name must be defined in the model
+};
 // Initialize properties
-Dictionary<string, object> props = new Dictionary<string, object>();
+var props = new Dictionary<string, object>();
 props.Add("active", true);
 rel.Properties = props;
-client.CreateOrReplaceRelationshipAsync("mySourceTwin", "rel001", rel);
+await client.CreateOrReplaceRelationshipAsync("mySourceTwin", "rel001", rel);
 // </CreateRelationship_short>
 
 // ------------------ CREATE RELATIONSHIP (Alternate) ---------------------
 // <CreateRelationship_3>
 // Create Twins, using functions similar to the previous sample
-await CreateRoom("Cafe", 70, 66);
-await CreateFloor("GroundFloor", averageTemperature=70);
+await CreateRoomAsync("Cafe", 70, 66);
+await CreateFloorAsync("GroundFloor", averageTemperature = 70);
```

Comment:
I'm not sure what `averageTemperate = 70` means as a parameter. This feels like nonsense to me, but as there is no impl of `CreateFloorAsync`, I don't know how to interpret it.

Reply:
Good catch! It's from [the graph concept doc](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-twins-graph#create-relationships) and it doesn't really make sense there either. 

@baanders any reason we can't use the CreateRelationship_short example from above here as well? That one is called in [this doc](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-use-apis-sdks#create-a-relationship)

We should probably add the try/catch block to both examples regardless, right?

Reply:
@alexkarcher-msft Sounds good to me--this doesn't seem to add much value as an alternate example (especially if we add the try/catch block to the other one as well), and the use of methods with no implementation could just be confusing.

I'm ok with removing this example and updating the doc to use CreateRelationship_short in its place.

Reply:
Switching out the example in the doc in PR [#145725](https://github.com/MicrosoftDocs/azure-docs-pr/pull/145725), which should go live at 3 PM PST today (2/4). After that, it's safe to remove CreateRelationship_3.